packages/shared/src/hooks/search/useSearchProviderSuggestions.ts

```diff
@@ -28,15 +30,16 @@ export const useSearchProviderSuggestions = ({
 }: UseSearchProviderSuggestionsProps): UseSearchProviderSuggestions => {
   const { user } = useAuthContext();
   const { getSuggestions } = useSearchProvider();
+  const debouncedQuery = useDebounce(query, defaultSearchDebounceMs);
```

Comment:
I would put it up at something a bit bigger, like 600ms, since 200ms is not doing anything for someone typing more leisurely or on mobile. We can even have different debounce on desktop/mobile.

Reply:
Yeah, 500 should be better as well. Only thing it will introduce a bit of latency for panel to show/update suggestions first but personally for me personally it is not an issue.

Reply:
It will actually hit on the first letter, then debounce the extra ones (due to query key debounce)

Thought it was a nice bonus 😂